arbitrator/prover/src/host.rs

```diff
@@ -286,6 +292,15 @@ impl Hostio {
                 opcode!(LocalGet, 1);
                 opcode!(ReadPreImage, PreimageType::Sha2_256);
             }
+            WavmReadHotShotCommitment => {
+                opcode!(LocalGet, 0);
```

Comment:
@nomaxg do you know what these `opcode` do, and why it's 0, 1 for the commitment and just 0 for the liveness?

Reply:
Here defines how the `WavmReadHotShotCommitment` should be converted into OpCodes. `LocalGet` means [getting something from somewhere and putting them to the `value_stack`](https://github.com/EspressoSystems/nitro-espresso-integration/blob/c7db7e868bae14f363b4ebba354a2129d8edfc3a/arbitrator/prover/src/machine.rs#L2137).
Since the `ReadHotShotCommitment` [opcode requires 2 argument](https://github.com/EspressoSystems/nitro-espresso-integration/blob/c7db7e868bae14f363b4ebba354a2129d8edfc3a/arbitrator/prover/src/machine.rs#L2487), so we have to `LocalGet` two times. (One pointer for the place where the commitment will be stored, and the other for the hotshot height)
As for the `Liveness`, we don't need to store the bool value into the module memory. (I don't quite understand why, but `readInboxMessage` stores the messages into the memory and the hotshot commitment seems something similar to the InboxMessage)